event-session-signup/README.md

```diff
@@ -0,0 +1,13 @@
+# Sign Up for Sessions at an Event
+Allows event attendees to sign up for sessions at an event and creates an itinerary.
+
+## Description
+Quickly create an end to end conference registration system. Start with the conference event information listed in a Google Sheet, then use Apps Script to create a sign-up form and a personalized itinerary for each attendee. For each attendee, a personalized schedule in a Google Doc is emailed out AND a conference calendar is created then populated with the specific sessions.
+
+## Set Up Instructions
+1. Create a copy of the sample [Event Session spreadsheet](https://docs.google.com/spreadsheets/d/1cpGsysprd5zl8VHYs4njsTSXwGxMA7DZPCW5o2EhR_8/copy)
+1. Click on custom menu item 'Conference' > 'Set up conference'
+1. A dialog box will appear and tell you that the script requires authorization. Read the authorization notice and 1. continue.
+1. The Conference menu will now be gone, but you will see a new Form menu instead. Click it and select Go to live form.
+1. Fill out the form
+1. Check your email for your personalized itinerary and check your Calendar on May 13th to see your registered sessions
```

Comment:
May 13th of what year?

Reply:
Should I actually make the date column a macro to today's date? but then does this mess up the data .csv file?
https://docs.google.com/spreadsheets/d/13RFXzUeE8pzT7IYlMopCdaFPM3b4oqRIIy5d0B0szYA/edit#gid=878320051

Reply:
the original quickstart has 2013 so I at least updated it to this year but using today's date seems better

Reply:
Using a dynamic date like  today + n days would be better. Point was more that if it's a hard coded date like May 13, somebody reading this 2 years from now will have a hard time finding it. Shouldn't make them look back to previous years in their calendar to find the event